packages/generator-single-spa/src/svelte/templates/setupTests.js

```diff
@@ -0,0 +1 @@
+import "@testing-library/jest-dom/vitest";
```

Comment:
Pretty weird that `@testing-library/jest-dom` includes an implementation for vitest, since jest and vitest are competitor libraries.